Calc/CustomViews/OperationBar.swift

```diff
@@ -22,8 +23,11 @@ class OperationBar: UIStackView {
     @IBOutlet weak var minusButton: UIButton!
     @IBOutlet weak var multiplyButton: UIButton!
     @IBOutlet weak var divideButton: UIButton!
+    @IBOutlet weak var powButton: UIButton!
+    @IBOutlet weak var rootButton: UIButton!
     @IBOutlet weak var swapButton: UIButton!
     
+    
     public var curentOperationName: NameOfOperation = NameOfOperation.plus
```

Comment:
Typo in currentOperationName, remove NameOfOperation
public var currentOperationName: NameOfOperation = .plus